net/socket/tcp_socket_starboard.cc

```diff
@@ -50,7 +55,32 @@ TCPSocketStarboard::~TCPSocketStarboard() {
 int TCPSocketStarboard::Open(AddressFamily family) {
   DCHECK_CALLED_ON_VALID_THREAD(thread_checker_);
   DCHECK(!SbSocketIsValid(socket_));
+#if SB_API_VERSION < 16
   socket_ = SbSocketCreate(ConvertAddressFamily(family), kSbSocketProtocolTcp);
+#else
+  int socket_fd = socket(ConvertSocketAddressTypeToDomain(ConvertAddressFamily(family)), 
```

Comment:
Is this just temporary till you get all the APIs adjusted?

Reply:
Yes.